src/task/index.html

```diff
@@ -106,7 +106,7 @@
             <option value="ASSIGNED">Assigned</option>
           </select>
         </div>
-        <div class="inputBox">
+        <div class="inputBox hidden">
```

Comment:
hiding this until we have the api

Reply:
Add a comment in the file, instead of in PR,
No one goes back and check PR review comments.

Reply:
@alishvagadiya I think the comment is meant for the reviewers of just this PR. It's probably there as an optional information.